Sources/Socket.swift

```diff
@@ -56,7 +56,8 @@ open class Socket: Hashable, Equatable {
             #if os(Linux)
                 return ntohs(addr.sin_port)
             #else
-                return Int(OSHostByteOrder()) != OSLittleEndian ? addr.sin_port.littleEndian : addr.sin_port.bigEndian
+                let sin_port = pointer.pointee.sin_port
```

Comment:
This should work, but you didn't fix the `os(Linux)` version All you need to do is move this `let sin_port ...` line up above the if statement :)

Reply:
Fixed.